src/amchart/CommonFunnel.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 (function(root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../common/HTMLWidget", "amcharts.funnel", "require"], factory);
+        define(["d3", "../common/HTMLWidget", "amcharts.funnel", "require", "amcharts.plugins.export"], factory);
     } else {
         root.amchart_CommonFunnel = factory(root.d3, root.common_HTMLWidget, root.AmCharts, root.require);
```

Comment:
This is the "non amd" factory ^^^^ you need to ensure its getting loaded in the non amd world as well...